src/data/defaults.txt

```diff
@@ -995,7 +995,9 @@ user	lastSlimeVial3893
 user	lastSlimeVial3894
 user	lastSlimeVial3895
 user	lastSlimeVial3896
+user	lastSelectedFaxbot
```

Comment:
I'd be OK if this defaulted to 0.  That suggests to the careful reader that this is an index and not a faxbot name,